aspnetcore/tutorials/getting-started-with-swashbuckle.md

```diff
@@ -3,15 +3,15 @@ title: Get started with Swashbuckle and ASP.NET Core
 author: zuckerthoben
 description: Learn how to add Swashbuckle to your ASP.NET Core web API project to integrate the Swagger UI.
 ms.author: scaddie
-monikerRange: '>= aspnetcore-3.1 < aspnetcore-8.0'
+monikerRange: '>= aspnetcore-3.1'
 ms.custom: mvc
 ms.date: 11/09/2021
 uid: tutorials/get-started-with-swashbuckle
 ---
 # Get started with Swashbuckle and ASP.NET Core
 
 > [!NOTE]
```

Comment:
Should this note be configured to only display for net8.0 and later?

Reply:
> Should this note be configured to only display for net8.0 and later?

Fixed